.github/workflows/npm.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Pack Testing
         run: ./scripts/npm-pack-testing.sh
         env:
-          WECHATY_PUPPET_SERVICE_TOKEN: ${{ secrets.WECHATY_PUPPET_SERVICE_TOKEN }}
+          WECHATY_PUPPET_SERVICE_TOKEN: ${{ secrets.WECHATY_PUPPET_SERVICE_TOKEN || 'puppet_test_testtoken' }}
```

Comment:
Please reverse this change. The tests need to use the environment variable for easy management.

Reply:
if we revert this, CI will always fail for PR from forked repos. See https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow